src/Query.ts

```diff
@@ -467,6 +468,9 @@ export const Query = {
     return new SelectQuery();
   },
   stats: () => new SelectQuery().from("(?)", "t"),
+  delete: (from: string, alias?: string) => new DeleteMutation(from, alias),
+  update: (table: string, alias?: string) => new UpdateMutation(table, alias),
+  insert: (into: string) => new InsertMutation(into),
   deserialize: SelectQuery.deserialize,
```

Comment:
can we please update the deserialize method to support mutations as well?

Reply:
Done. I moved it into a separate function to make it cleaner.